### DISCUSSION TITLE
What is the 💴 of 🇲🇾?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1076] [Galaxy Trivia] [answered]
**Bonus:** What is the largest city in 🇦🇹?

**Comment by NomanGhiasy** [comment]
Ringgit

**Comment by peterbaccaro** [comment]
The currency of Malaysia is the **Malaysian ringgit (MYR)**.

The largest city in Austria is **Vienna**. Vienna, also known as Wien in German, is the capital and the primary cultural, economic, and political centre of Austria. It is the most populous city in the country and serves as one of the nine federal states of Austria. Vienna is renowned for its historical heritage, classical music, art, architecture, and rich cultural scene.

**Comment by thuongtruong1009** [comment, marked as answer]
The 💴 of 🇲🇾 is the Malaysian Ringgit (MYR).

The largest city in 🇦🇹 (Austria) is Vienna (Wien).
